packages/workspace/src/browser/workspace-commands.ts

```diff
@@ -221,17 +221,20 @@ export class FileSystemCommandHandler implements CommandHandler {
         protected readonly handler: UriCommandHandler
     ) { }
 
-    protected getUri(): URI | undefined {
+    protected getUri(...args: any[]): URI | undefined {
+        if (args && args[0] instanceof URI) {
+            return args[0];
+        }
         return UriSelection.getUri(this.selectionService.selection);
     }
 
     execute(...args: any[]): object | undefined {
-        const uri = this.getUri();
+        const uri = this.getUri(...args);
```

Comment:
Is this used ? same for the other commands.. I don't see a uri in arg 0 anywhere?

Reply:
It is in git-widget.ts:222.

Reply:
Ha OK thx